src/app/Components/ArtworkRail/ArtworkRailCard.tsx

```diff
@@ -216,14 +173,14 @@ export const ArtworkRailCard: React.FC<ArtworkRailCardProps> = ({
           testID={testID}
         >
           <Flex backgroundColor={backgroundColor}>
-            <ArtworkRailCardImage
-              containerWidth={containerWidth}
-              image={image}
-              urgencyTag={!displayAuctionSignal ? urgencyTag : null}
-            />
+            {enableArtworkRailRedesignImageAspectRatio ? (
+              <ArtworkRailCardImage artwork={artwork} />
+            ) : (
+              <LegacyArtworkRailCardImage artwork={artwork} urgencyTag={urgencyTag} />
```

Comment:
This is emerald specific, can we remove the urgency tag since they're supposed to be removed after releasing the new auction signals?

```suggestion
              <LegacyArtworkRailCardImage artwork={artwork} urgencyTag={!displayAuctionSignal ? urgencyTag : null} />
```

Reply:
This is great! So I can remove all the logic around urgency tags within the artwork rail completely?

Reply:
since we released the auction signals, yeah feel free to remove the urgency tags logic :)

Reply:
👍